api-tests/platform/targets/tgt_dev_apis_stdc/target.cmake

```diff
@@ -44,10 +90,16 @@ if(${SUITE} STREQUAL "CRYPTO")
 	)
 endif()
 if(${SUITE} STREQUAL "PROTECTED_STORAGE")
-	message(FATAL_ERROR "Protected Storage not supported")
+	list(APPEND PAL_SRC_C_NSPE
+		${PSA_ROOT_DIR}/platform/targets/${TARGET}/nspe/protected_storage/pal_protected_storage_intf.c
+	)
+	_create_psa_storage_exe(psa-arch-tests-ps protected_storage)
```

Comment:
Can similar binary support for crypto added as part of this PR?

Reply:
Added as requested.